docs/guide/3-install-free5gc.md

```diff
@@ -151,10 +151,10 @@ sudo systemctl disable ufw # prevents the firewall to wake up after a OS reboot
 1. Before building WebConsole, install nodejs and yarn packages first:
 
     ```
-    sudo apt remove cmdtest
-    sudo apt remove yarn
+    sudo apt remove yarn cmdtest
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
```

Comment:
Since corepack is installed by default, we can just `corepack enable` to setup Yarn.

```suggestion
```

Reply:
First of all, thanks for testing the WebConsole with NodeJS v20, I committed your suggestion.

I've kept these commands regarding yarn because it was working and the command you suggested (corepack) wasn't installed on Ubuntu 20.04 even after setting up yarn via APT (screenshot attached below). What did I do wrong?

![image](https://github.com/free5gc/free5gc.github.io/assets/25849028/19a741a0-02fa-4739-acbf-65dee33d059a)

TIA